inst/dvhydrograph/dvhydrograph.Rmd

```diff
@@ -20,15 +20,21 @@ cat(getLogo())
 
 `r if(!isEmptyOrBlank(data[['reportMetadata']][['primaryDescriptions']])) {paste("**Primary Time Series:**", data[['reportMetadata']][['primaryDescriptions']])}`
 
-`r if(!isEmptyOrBlank(data[['reportMetadata']][['downChainDescriptions1']])) {paste("**First Down Chain Time Series:**", data[['reportMetadata']][['downChainDescriptions1']])}`
+`r if(!isEmptyOrBlank(data[['reportMetadata']][['downChainDescriptions1']])) {paste("**Stat Derived Time Series 1:**", data[['reportMetadata']][['downChainDescriptions1']])}`
 
-`r if(!isEmptyOrBlank(data[['reportMetadata']][['downChainDescriptions2']])) {paste("**Second Down Chain Time Series:**", data[['reportMetadata']][['downChainDescriptions2']])}`
+`r if(!isEmptyOrBlank(data[['reportMetadata']][['downChainDescriptions2']])) {paste("**Stat Derived Time Series 2:**", data[['reportMetadata']][['downChainDescriptions2']])}`
 
-`r if(!isEmptyOrBlank(data[['reportMetadata']][['downChainDescriptions3']])) {paste("**Third Down Chain Time Series:**", data[['reportMetadata']][['downChainDescriptions3']])}`
+`r if(!isEmptyOrBlank(data[['reportMetadata']][['downChainDescriptions3']])) {paste("**Stat Derived Time Series 3:**", data[['reportMetadata']][['downChainDescriptions3']])}`
 </div>
 <div class="header-col-right">
 **Period:**  `r as.Date(data[['reportMetadata']][['startDate']])` to `r as.Date(data[['reportMetadata']][['endDate']])`
 
+`r if(!isEmptyOrBlank(data[['reportMetadata']][['inputDataDescriptions2']])) {paste("**Secondary Reference Time Series:**", data[['reportMetadata']][['inputDataDescriptions2']])}`
+
+`r if(!isEmptyOrBlank(data[['reportMetadata']][['inputDataDescriptions3']])) {paste("**Tertiary Reference Time Series:**", data[['reportMetadata']][['inputDataDescriptions3']])}`
+
+`r if(!isEmptyOrBlank(data[['reportMetadata']][['inputDataDescriptions4']])) {paste("**Quaternary Reference Time Series:**", data[['reportMetadata']][['inputDataDescriptions4']])}`
+
```

Comment:
That's a good point. Changes would need to be made on the UI too in the advanced settings cog. Might be worth asking Laura?